setup.py

```diff
@@ -80,6 +80,7 @@ def package_vars(version_file):
             "coverage < 5.0",
             "pytest-cov",
             "pytest",
+            "semver",
```

Comment:
`semver` isn't needed for testing, is it?  I think it fits better as a "dev" extra, along with ipython.

Reply:
That makes sense. Changes made in e3e3d90.